tmt/steps/prepare/feature/fips-disable.yaml

```diff
@@ -0,0 +1,19 @@
+- hosts: all
+  user: root
+  tasks:
+    # RHEL 7 or CentOS 7
+    - when: ansible_distribution_major_version == "7"
+      block:
+        - name: Enable fips
+          command: echo UNSUPPORTED
+
+    # RHEL 8          or RHEL 9
+    # CentOS Stream 8 or CentOS Stream 9
+    - when: ansible_distribution_major_version == "8" or ansible_distribution_major_version == "9"
```

Comment:
Let's make it future ready - there will be rhel-10 (and more?) in the future.

Reply:
Hm, `when: ansible_distrobution_major_version|int >= 8` should work then.

Reply:
Thanks, updated.